### PR TITLE
Show worker UUID and handle heartbeat exception

### PR DESCRIPTION
**Pull request by tomtor**
See worker_info["unique_key"]

### CODE REVIEW

**Comment by tomtor** [comment]
@ppigazzini Did not squash because these two fixes are unrelated. Note that version bump is included in the second commit.

**Comment by tomtor** [comment]
See https://github.com/glinscott/fishtest/issues/851#issuecomment-723432778

@d3vv

**Comment by ppigazzini** [comment]
@tomtor DEV updated w/ #857 and #860, workers updated w/ #860, workers show:
```
Score of New-3f6451eff7 vs Base-3f6451eff7: 3 - 3 - 0  [0.500] 6
Started game 9 of 200 (New-3f6451eff7 vs Base-3f6451eff7)
Finished game 7 (New-3f6451eff7 vs Base-3f6451eff7): 0-1 {Black wins by adjudication}
Score of New-3f6451eff7 vs Base-3f6451eff7: 3 - 4 - 0  [0.429] 7
Started game 10 of 200 (Base-3f6451eff7 vs New-3f6451eff7)
Finished game 10 (Base-3f6451eff7 vs New-3f6451eff7): 1-0 {White wins by adjudication}
Score of New-3f6451eff7 vs Base-3f6451eff7: 3 - 5 - 0  [0.375] 8
Started game 11 of 200 (New-3f6451eff7 vs Base-3f6451eff7)
Finished game 9 (New-3f6451eff7 vs Base-3f6451eff7): 1-0 {White wins by adjudication}
Score of New-3f6451eff7 vs Base-3f6451eff7: 4 - 5 - 0  [0.444] 9
Exception from calling update_task:
Expecting value: line 1 column 1 (char 0)
Send heartbeat for 8bff0b82-d540-4b52-b642-dffed6ced0ef ... Pleased to hear from you...
Exception from calling update_task:
Expecting value: line 1 column 1 (char 0)
Exception from calling update_task:
Expecting value: line 1 column 1 (char 0)
```

**Comment by d3vv** [comment]
Could it possible add as additional a worker name for such debug? Name like `dew-8cores-12` for example..

**Comment by d3vv** [comment]
Talking about "my UID" I mean a Worker Name recently :)) Sure "worker name" more useful than mega-true uuid as it used on fishtest web-site from time immemorial  : )

**Comment by tomtor** [comment]
> Could it possible add as additional a worker name for such debug? Name like `dew-8cores-12` for example..

That is a server side name. I could return that as heartbeat response.

**Comment by tomtor** [comment]
@d3vv https://github.com/glinscott/fishtest/pull/857/commits/23bfd1728ed1420fa735f822f5d6c3be20fdc785

so it shows:

```
Send heartbeat for 75a6cd41-dca0-4376-8c33-11a4a057c21b ... user00-3cores
```

File: worker/worker.py [hunk 1]
```diff
@@ -125,23 +125,27 @@ def get_rate():
...
+        "unique_key": worker_info["unique_key"],
+    }
     count = 0
     while ALIVE:
         time.sleep(1)
         count += 1
         if count == 60:
             count = 0
-            print("Send heartbeat... ", end="")
+            print("Send heartbeat for", worker_info["unique_key"], end=" ... ")
```
Comment (ppigazzini) [inline worker/worker.py:139]: It's a bit hard to read the heartbeat message now, I suggest to indent with two space:

```
Finished game 153 (New-3f6451eff7 vs Base-3f6451eff7): 1-0 {White wins by adjudication}
Score of New-3f6451eff7 vs Base-3f6451eff7: 49 - 49 - 55  [0.500] 153
Started game 156 of 200 (Base-3f6451eff7 vs New-3f6451eff7)
  Send heartbeat for 695867dc-48a1-434d-8269-36edf45ee0f8 ... user00-3cores-16
Finished game 155 (New-3f6451eff7 vs Base-3f6451eff7): 0-1 {Black wins by adjudication}
Score of New-3f6451eff7 vs Base-3f6451eff7: 49 - 50 - 55  [0.497] 154
```

Comment (tomtor) [reply worker/worker.py]: Done...

**Review by ppigazzini — APPROVED** [review]

**Comment by ppigazzini** [comment]
PROD updated, thank you @tomtor :)

**Pull request merged**